src/styles/app-header/app-header.css

```diff
@@ -2,6 +2,9 @@
   background: var(--color-primary);
   color: var(--color-white);
   padding: 12px;
+  position: fixed;
+  top: 0;
+  width: 600px;
```

Comment:
You added a grid to your `body` with the property: `grid-template-rows: 48px auto 60px;` which means that you create 3 rows for the page layout.
When you have that, you don't need the `position: fixed;`, if your HTML structure is as followed:
```
<body>
  <header>....</header>
  <main>...</main>
  <footer>...</footer>
</body>
```

This way the first row will be the header, the second the main and the third the footer. We can also discuss this in the course if it is not clear.

Reply:
Fixed it. was a thinking fail :)